src/core/qgsdistancearea.h

```diff
@@ -222,6 +222,21 @@ class CORE_EXPORT QgsDistanceArea
      */
     double measureLine( const QgsPoint &p1, const QgsPoint &p2, QgsUnitTypes::DistanceUnit &units ) const;
 
+    /**
+     * calculates distance from one point with distance in meters and azimuth (direction)
+     * based on inverse Vincenty's formula when Geographic
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * note: usage:
+     *  QgsDistanceArea myDa;
+      * myDa.setSourceCrs( layer->crs() );
+     * \since QGIS 3.0
+     * @param p1 start point [can be Cartesian or Geographic]
```

Comment:
Don't worry too much about it - we're moving everything to the "sipify" script which auto creates the sip bindings directly from headers, and part of that process autoconverts the doxygen notes to Python docstrings. The doxygen comments in unsipified .sip files has no effect. So when qgsdistancearea.h is sipified the docs will automatically get fixed.